nautobot_circuit_maintenance/templates/nautobot_circuit_maintenance/note.html

```diff
@@ -46,8 +46,8 @@ <h1>{{ object.title }}</h1>
             <td>{{ object.comment }}</td>
         </tr>
         <tr>
-            <td>Date</td>
-            <td>{{ object.date }}</td>
+            <td>Last Updated</td>
```

Comment:
Not blocking for this PR, but it did make me notice that although we're using change-logged base classes for all of these models, our page templates don't provide a tab for viewing the changelog. (You can see that the changelog exists from the Nautobot home page summary, though.) We should fix that.

(When/if we move to requiring Nautobot 1.2, this will be made a lot easier by the implementation of https://github.com/nautobot/nautobot/pull/737)

Reply:
@glennmatthews , do you mean about [this](https://github.com/nautobot/nautobot-plugin-circuit-maintenance/blob/develop/nautobot_circuit_maintenance/templates/nautobot_circuit_maintenance/circuitmaintenance.html#L39-L43)?
is it only supported for `CircuitMaintenance` and `CircuitImpact`. I would add an issue to update the other in a different PR

Reply:
Yeah, that part of the template. I guess the main one I noticed it missing from was `Note`. I'll open an issue.